build.sbt

```diff
@@ -717,11 +717,20 @@ lazy val scioTensorFlow: Project = Project(
 lazy val scioSchemas: Project = Project(
   "scio-schemas",
   file("scio-schemas")
-).enablePlugins(ProtobufPlugin)
-  .settings(commonSettings)
+).settings(commonSettings)
   .settings(noPublishSettings)
-  .settings(protobufSettings)
   .settings(
+    PB.protocVersion := "-v371",
+    PB.targets in Compile := Seq(
+      PB.gens.java -> (sourceManaged in Compile).value,
+      scalapb.gen(javaConversions = true) -> (sourceManaged in Compile).value
+    ),
+    libraryDependencies ++= Seq(
+      "io.grpc" % "grpc-netty" % grpcVersion,
+      "com.thesamet.scalapb" %% "scalapb-runtime-grpc" % scalapb.compiler.Version.scalapbVersion
```

Comment:
Do we REALLY need ScalaPB here? Can't use the gPRC native java proto?

Reply:
Yes we can use the gPRC-java native plugin, the idea was to provide the Scala example without converting futures from Java to Scala.